blockstore/src/test/yaml/def-suff-b/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
@aledsage @murdoaird @nakomis @iyovcheva Isn't it better if we use Cloudsoft Developer License?